docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md

```diff
@@ -32,52 +34,58 @@ For more information on our GitHub organizations and their uses, see [B.C. gover
 
-## Enterprise account ownership
+GitHub Enterprise offers various features beyond those of the Free and Teams tiers. To find out more about the features of GitHub Enterprise, you may wish to review [GitHub's product page](https://github.com/enterprise).
 
-The GitHub Enterprise account is owned by OCIO and managed by the Platform Services team. Ministries own GitHub organizations linked to the B.C. government’s GitHub Enterprise accounts and the private repositories within their organizations.  
+There are certain notable differences between the [public facing `bcgov` GitHub organization](https://github.com/bcgov) and B.C. government's GitHub Enterprise environment. For example, the ability to create public repositories is disabled within GitHb Enterprise since with B.C. government public open source code should be in the [public `bcgov` organization](https://github.com/bcgov). 
 
-User licenses for accessing the private repositories come from a central pool under the GitHub Enterprise account and must be purchased for each user that needs access to the private repositories in the ministry’s private organizations.  Management of this pool of seats is done in partnership with each sector or ministry purchasing their own allocated seat count.
+Other details of GitHub Enterprise include the following:
 
-When your private organization is linked to the GitHub Enterprise account certain settings are automatically enabled or disabled. For example, the ability to create public repositories within this private organization is removed to fall in line with all Provincial open-source code remaining in `bcgov`. Other features include the following:
+* Single sign-on is provided via Azure AD (aka Entra ID)
+* Login uses government-issued IDIR identities
+* Users will be automatically assigned new GitHub IDs, which will change the login experience for users working in both GitHub.com repositories and B.C. government GitHub Enterprise 
+* GitHub Actions is enabled
+* Code Dependency Insights is enabled
 
-* Two-factor authentication is mandatory
-* GitHub Actions is enabled for all
-* Code Dependency Insights is enabled for all organizations linked to the GitHub Enterprise account
+## GitHub Enterprise account ownership
 
-## Request GitHub Enterprise user licences
+The B.C. government's GitHub Enterprise account is managed centrally by the Digital Office's [Developer Experience Team](mailto:developer.experience@gov.bc.ca). Ministries or sectors are provided with GitHub "organizations" within the central GitHub Enterprise account. Teams' private repositories are created within these organizations.  Management of ministry or sector organizations within GitHub Enterprise is delegated to staff within the corresponding IMB/ISB. 
 
-Contact Software Central Management at SoftwareCentral.Management@gov.bc.ca and cc Dean Picton (Dean.Picton@gov.bc.ca). Use the subject line "GitHub Enterprise Request" and send the following information:  
```

Comment:
@sheaphillips @jenreiher-bcgov The sentence "Use the subject line "GitHub Enterprise Request" and send the following information:" has a slightly different recommended email subject from Steps. Also, as no following information is available here, is that better to say something like "Please refer to the "Steps" section below."? What do you think?